packages/web-components/src/components/ic-radio-option/ic-radio-option.tsx

```diff
@@ -76,10 +81,14 @@ export class RadioOption {
   }
 
   private defaultRadioValue: string = "";
+  private skipFocus = false;
```

Comment:
Just so I understand better, why do we have a skipFocus flag? Was there initial a focus state on the radio button when reset was pressed?

Reply:
We are changing the state of selected which invokes the handleClick by way of the watch on the selected prop. When this happens the radio receives focus, but we do not want it to in this case